src/english.yml

```diff
@@ -214,6 +214,7 @@ msg_unable_ally_npc: "&cUnable to request an alliance because the mayor of the t
 
 msg_claimed: '&2Successfully claimed'
 msg_unclaimed: '&2Successfully unclaimed'
+msg_claim_error: '&cAnother plugin stopped the claim of (%d)/(%d) town blocks, could not complete the operation.'
```

Comment:
This should go at the end of the file, the language file should have it's version number bumped by one, and for consistency's sake the %d's should be %s.